src/common/utility.js

```diff
@@ -62,7 +62,8 @@ const checkElemInArray = (tab_list, tab) => tab_list.includes(tab)
 
 const getWindowWidth = () => (isBrowser() && window.screen ? window.screen.width : '')
 
-const getLanguage = () => (isBrowser() ? localStorage.getItem('i18n') || navigator.language : null)
+const getLanguage = () =>
```

Comment:
this will break _signup-new , as it needs `-` for redirecting the language, lets not change it in utility, we can use replace function specifically in socket_base, or make another function specifically to replace the case and use it in socket base only

Reply:
Yes, I just realised this will break the `/${getLanguage()}/terms-and-conditions/` redirection rule. Will fix right away.